operator-framework-quarkus-extension/runtime/src/main/java/io/javaoperatorsdk/quarkus/extension/ExternalControllerConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package io.javaoperatorsdk.quarkus.extension;
+
+import io.javaoperatorsdk.operator.api.config.ControllerConfiguration;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import java.util.List;
+import java.util.Optional;
+
+@ConfigGroup
+public class ExternalControllerConfiguration {
+
+  /**
+   * An optional list of comma-separated namespace names the controller should watch. If the list
+   * contains {@link ControllerConfiguration#WATCH_ALL_NAMESPACES_MARKER} then the controller will
+   * watch all namespaces.
+   */
+  @ConfigItem public Optional<List<String>> namespaces;
```

Comment:
How about if this list is empty then the controller watches all namespaces? I think watching all namespaces is a sensible default as this will mostly be controlled by RBAC anyway by cluster admins not by configuring the operator.

Reply:
At the moment the default behavior is to watch the namespace the client is connected to (which should be the namespace the operator is deployed in).

Reply:
Ok let's leave it like that for now. We could check what's the default in the operator-sdk and just run with that later.

Reply:
I will address this in #303